src/rp2_common/cmsis/include/cmsis/rename_exceptions.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2020 Raspberry Pi (Trading) Ltd.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef _CMSIS_RENAME_EXCEPTIONS_H
+#define _CMSIS_RENAME_EXCEPTIONS_H
+
+// PICO_CONFIG: PICO_CMSIS_RENAME_EXCEPTIONS, Whether to rename SDK exceptions such as isr_nmi to their CMSIS equivalent i.e. NMI_Handler, type=bool, default=0, group=cmsis_core
```

Comment:
This line says `default=0`, but line 14 seems to set the default to 1? :confused:

Reply:
please open a PR

Reply:
What's the intended behaviour - change the PICO_CONFIG default to 1, or change the actual #define to 0 ?

Reply:
yeah the #define is correct i.e. 1